scripts/dockerbot/check_prs.py

```diff
@@ -23,6 +25,31 @@
 
 logger = create_logger('check_prs', 'prt.log')
 
+# Disable pika logs
+logging.getLogger("pika").propagate = False
+
+
+def send_message_to_bot(msg):
+    if 'rabbitmq_url' not in docker_conf.key():
+        logger.warn("Cannot send message to RabbitMQ - rabbitmq_url is missing in docker.yaml")
+        return
+
+    url = docker_conf['rabbitmq_url']
+    params = pika.URLParameters(url)
+    params.socket_timeout = 5
+    connection = pika.BlockingConnection(params)  # Connect to CloudAMQP
+    try:
+        channel = connection.channel()
+        channel.queue_declare(queue='hello')
+        message = {"channel": "#cfme-qe", "message_type": "github", "body": msg}
+        channel.basic_publish(exchange='', routing_key='hello',
+                              body=json.dumps(message, ensure_ascii=True))
+    except Exception:
+        output = traceback.format_exc()
+        logger.warn("Exception while sending a message to the bot: {}".format(output))
+    finally:
+        connection.close()
```

Comment:
We'll miss the logging in this case (or we'll have to reraise the exception and log it, which doesn't add any value)